src/main/java/rip/bolt/ingame/team/ConfigTeamParser.java

```diff
@@ -66,6 +71,11 @@ public TournamentTeam getTeam(String name) {
         return teams;
     }
 
+    public int reloadTeams() {
+        teams = parseTournamentTeams(new File(Tournament.get().getDataFolder(), "teams"));
```

Comment:
```suggestion
        teams = parseTournamentTeams(new File(Tournament.get().getDataFolder(), "teams"), new File(Tournament.get().getDataFolder(), "teams.yml"));
```
Your code at the moment doesn't reload teams.yml

Reply:
I don't like the hardcoded teams.yml at all, should really be an optional parameter in /tm reload <filename> to only load specific team files. I could add that.

Reply:
It should reload all teams files. Not reloading teams.yml (or having a flag) isn't intuitive.